conf/fileserver.conf

```diff
@@ -27,11 +27,44 @@
 
 # PERMISSIONS
 
-# Every static mount point should have an `allow *` line; setting more
-# granular permissions in this file is deprecated. Instead, you can
-# control file access in auth.conf by controlling the
-# /file_metadata(s)/<mount point> and /file_content(s)/<mount point> paths.
-# For example:
+# Setting permissions - for example, using the allow, allow_ip, or deny
+# directives - is no longer supported. Instead, you can control file access in
```

Comment:
`deprecated` seems to imply that it still works but that we're just discouraging the use of it whereas in this case we've actually removed the functionality.  Would something like this be better?

> The ability to set permissions - for example, using the allow, allow_ip, or deny directives - has been removed.